__tests__/endpoints.test.js

```diff
@@ -41,3 +41,44 @@ describe("Get /api", () => {
       });
   });
 });
+
+describe("GET /api/articles/:article_id", () => {
+  test(`Responds with:
+
+  an article object, which should have the following properties:
+  author
+  title
+  article_id
+  body
+  topic
+  created_at
+  votes
+  article_img_url`, () => {
+    return request(app)
+      .get("/api/articles/3")
+      .expect(200)
+      .then(({ body }) => {
+        const { article } = body;
+        
+        expect(article).toMatchObject({
+          article_id: expect.any(Number),
+          title: expect.any(String),
+          topic: expect.any(String),
+          author: expect.any(String),
+          body: expect.any(String),
+          created_at: expect.any(String),
+          votes: expect.any(Number),
+          article_img_url: expect.any(String),
+        });
+      });
```

Comment:
We need to be a lot more specific for this test. Since we are wanting to get a specific article, we know what it's article_id is, what title we are expecting, who the author is etc.